code/src/pages/cart/components/empty-cart/EmptyCart.scss

```diff
@@ -0,0 +1,23 @@
+@import "@design-system";
+
+.empty-cart {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: spacing(10);
+
+  &__title {
+    padding-top: spacing(5);
+    padding: spacing(5);
+    color: var(--spa-empty-cart-color, $gray);
+    font-size: $fs-h3;
```

Comment:
Take advantage of using `AppTypography` here

Reply:
Fixed